Makefile

```diff
@@ -0,0 +1,24 @@
+INCLUDES= checkers
+
```

Comment:
Can you add a .PHONY here for the targets that don't actually make files?

Reply:
tweaked!